.github/workflows/build-pr-preview.yml

```
@@ -137,7 +137,7 @@ jobs:
         id: unified_docs_preview_url
         # save preview url from deployment and save copy of the preview url w/o https to fetch the deployment via Vercel REST API in next step
         run: |
-          preview_url=$(vercel deploy --prebuilt --target=preview --token=${{ secrets.VERCEL_TOKEN }})
+          preview_url=$(vercel deploy --prebuilt --target=preview --token=${{ secrets.VERCEL_TOKEN }}) --archive=tgz
```

I think the archive=tgz tag should be within the parentheses
```suggestion
          preview_url=$(vercel deploy --prebuilt --target=preview --token=${{ secrets.VERCEL_TOKEN }} --archive=tgz)
```

---

Woah you're totally right, it should!